modules/user-guide/pages/installing-vale-cli.adoc

```diff
@@ -1,49 +1,60 @@
 // Metadata for Antora
-:navtitle: Installing Vale by cloning the *{repository-name}* repo
+:navtitle: Installing Vale
 :keywords: vale
-:page-aliases: end-user-guide:using-vale-cli.adoc
-:description: Describes how to install and configure the Vale CLI
+:description: Describes how to install from the latest release and configure the Vale CLI
+:page-aliases: end-user-guide:using-vale-cli.adoc, installing-vale-cli-from-zip.adoc
 // End of metadata for Antora
-
+// Metadata for Modular Docs
 :context: assembly_getting-started-with-vale
 :_module-type: PROCEDURE
-[id="proc_using-vale-cli-on-a-local-environment_{context}"]
-= Installing and configuring Vale by cloning the *{repository-name}* repo
-
-To use `vale` CLI on any project in your local environment, install the `vale` CLI and register the *{repository-name}* Vale configuration as the default.
-
-NOTE: Alternatively, if you do not intend to contribute to the *{repository-name}* project, you can configure the `RedHat` Vale style by downloading the latest release from GitHub. See xref:installing-vale-from-zip.adoc[]
-
-.Prerequisites
+// End of metadata for Modular Docs
+[id="proc_using-vale-cli_{context}"]
+= Installing Vale with the `RedHat` package
 
-* The `git` tool is installed.
-* The `asciidoctor` tool is installed. See link:https://docs.asciidoctor.org/asciidoctor/latest/install/[Installing AsciiDoctor]
+To use Vale CLI with the `RedHat` style on any project without further configuration in the projects: install and configure the Vale CLI.
 
 .Procedure
 
-. Install the Vale command-line tool on your workstation. See link:https://docs.errata.ai/vale/install[Installing Vale].
+. Install the Vale command-line tool on your workstation.
+See link:https://vale.sh/docs/vale-cli/installation/[Installing Vale].
 +
-NOTE: On the Vale site, click the tab for your operating system.
+[NOTE]
+On the Vale site, click the tab for your operating system.
 
-. Clone the link:{repository-url}[{repository-name} repository] in your local environment.
+. Register the `RedHat` Vale package in the `.vale.ini` Vale configuration file. Save the file in your `$HOME` directory:
```

Comment:
When I tested this procedure,I already had a preconfigured .vale.ini in my $home folder. It got me thinking that another use case is:  Switching to packages when you have already been using Vale for some time.